gui-libs/libgedit-tepl/libgedit-tepl-6.10.0.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson virtualx
+
+DESCRIPTION="GtkSourceView-based text editors and IDE helper library"
+HOMEPAGE="https://github.com/gedit-technology/libgedit-tepl"
+SRC_URI="https://github.com/gedit-technology/${PN}/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="LGPL-3+"
+SLOT="6/4"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="gtk-doc"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	!gui-libs/tepl
+	>=dev-libs/glib-2.74:2
+	>=x11-libs/gtk+-3.22:3[introspection]
+	>=gui-libs/libgedit-gtksourceview-299.1.0:300
+	>=gui-libs/libgedit-amtk-5.0:5=[introspection]
+	gui-libs/libgedit-gfls
+	dev-libs/icu:=
+	gnome-base/gsettings-desktop-schemas
+
+	dev-libs/gobject-introspection:=
```

```suggestion
	gnome-base/gsettings-desktop-schemas
	dev-libs/gobject-introspection:=
```
I guess there is no reason to have an empty line here?

---

Removed empty line in https://github.com/gentoo/gentoo/commit/7384e7dadd374a1ef818cb1f078c4b2a7579974d